src/sentry/tasks/unmerge.py

```diff
@@ -470,15 +468,34 @@ def update_tag_value_counts(id_list):
         )
 
 
+def lock_hashes(project_id, source_id, fingerprints):
+    with transaction.atomic():
+        eligible_hashes = list(
```

Comment:
any reason to simply not do all of this in the same request? is it to avoid when any hash is flipped form active to inactive during it?